web/html/src/manager/minion/ansible/accordion-path-content.tsx

```diff
@@ -0,0 +1,182 @@
+import * as React from "react";
+import { AnsiblePath } from "./ansible-path-type";
+import Network from "utils/network";
+import { Messages, Utils } from "components/messages";
+import { Loading } from "components/utils/Loading";
+import { AceEditor } from "components/ace-editor";
+
+type PropsType = {
+  path: AnsiblePath;
+};
+
+type StateType = {
+  open: boolean; 
+  content: any;
```

Comment:
I'm guessing this is actually `PlaybookDetails[] | null`?

Reply:
Actually `content` depending on the `pathType` can be a `PlaybookDetails[]` if the type is `playbook` but also a `YAML` formatted `String` if the type is `inventory`, and also it can be `null` if it is not yet assigned.

```

Reply:
I think anything that offers information beyond `any` is good, yes. With `any` we don't get any benefits of type checking.